OxfordBodleian/Juel-Jensen/BDLaethd19.xml

```diff
@@ -224,9 +224,196 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                <desc>Full-page miniature with four panels, each with a single portait of:
+                                  <persName ref="PRS13683Asaph">Asaph</persName>
+                                  <persName ref="PRS13684Ethan">Ethan</persName>, and two sons of Korah.
+                                  <q xml:lang="gez"><persName ref="PRS13683Asaph">አሳፍ</persName></q>
+                                  <q xml:lang="gez"><persName ref="PRS13684Ethan">ኤታን፡</persName></q>
+                                   <q xml:lang="gez">ደቂቀ፡ ቆሬ፡</q>
+                                   <q xml:lang="gez">ደቂቀ፡ ቆሬ፡</q>
+                                </desc>
+                            </decoNote>
+
+                            <decoNote type="miniature" xml:id="d3">
+                                <locus target="#6r"/>
+                                <desc>Full-page miniature with six panels, each with a pair of portraits showing:
+                                  <persName ref="PRS3337Daniel">Daniel</persName> and
+                                  <persName ref="PRS5680Jeremiah">Jeremiah</persName>,
+                                  <persName ref="PRS4400Gad">Gad</persName> and
+                                  <persName ref="PRS13685Nathan">Nathan</persName>,
+                                  <persName ref="PRS3814Enoch">Enoch</persName> and
+                                  <persName ref="PRS3721Elijah">Elijah</persName>,
+                                  <persName ref="PRS3723Elisha">Elisha</persName> and
+                                  <persName ref="PRS5773Joshua">Joshua the Judge</persName>,
+                                  <persName ref="PRS8854Simeon">Simeon</persName> and
+                                  <persName ref="PRS5584Isaiah">Isaiah</persName>,
+                                  an unidentified figure (Zerubabel?) and
+                                  <persName ref="PRS3943Ezra">Second Ezra.</persName>
+
+                                  <q xml:lang="gez"><persName ref="PRS3337Daniel">ዳንኤል፡</persName> ነቢይ፡</q>
+                                  <q xml:lang="gez"><persName ref="PRS5680Jeremiah">ኤርምያስ፡</persName></q>
+                                  <q xml:lang="gez"><persName ref="PRS4400Gad">ጋድ፡</persName></q>
+                                  <q xml:lang="gez"><persName ref="PRS13685Nathan">ናታን</persName></q>
+                                  <q xml:lang="gez"><persName ref="PRS3814Enoch">ሄኖክ፡</persName></q>
+                                  <q xml:lang="gez"><persName ref="PRS3721Elijah">ኤልያስ፡</persName></q>
+                                  <q xml:lang="gez"><persName ref="PRS3723Elisha">ኤልሳዕ፡</persName></q>
+                                  <q xml:lang="gez"><persName ref="PRS5773Joshua">ኢያሱ፡ መስፍን።</persName></q>
+                                  <q xml:lang="gez"><persName ref="PRS5584Isaiah">ኢሳይያስ፡</persName></q>
+                                  <q xml:lang="gez">ዘሩ[…]ል።</q>
```

Comment:
Is the portion of text marked with [...] illegible due to faded ink or material damage?

Reply:
Damage

Reply:
Good, thank you. I took the freedom to encode it as `<gap reason="lost"/>`